lib/attributor/types/class.rb

```diff
@@ -0,0 +1,32 @@
+require_relative '../exceptions'
+
+module Attributor
+  class Class
+    include Type
+
+    def self.native_type
+      return ::Class
+    end
+
+    def self.load(value, context=Attributor::DEFAULT_ROOT_CONTEXT, **options)
+      unless value.kind_of?(::String) || value.nil?
+        raise IncompatibleTypeError,  context: context, value_type: value.class, type: self
+      end
+
+      value = "::" + value if value[0..1] != '::'
+
+      value && eval(value)
```

Comment:
what do you mean? I thought that was exactly the point of this. Have the attribute return the right Class object when loaded...not a String.

Many times you'll know what class you expect, so the `.of(X)` would allow  you to do that and get the right type in the output.

What I wrote above is that you could still allow an "any" class by not using the .of  ... and it is up to the user to allow that or not.